tools/golang/docker-compose.yml

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 ---
-version: '2.4'
+version: '2.3'
```

Comment:
why the downgrade?

Reply:
Because the version of docker engine available from "dockerrepo" on CentOS7 isn't compatible with `docker-compose.yml` version 2.4 - and the contents are perfectly compatible with 2.3 (apparently).